.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2.1
+
+orbs:
+  buildevents: honeycombio/buildevents@0.2.6
```

Comment:
if this is a public repo then we can't use buildevents

Reply:
This is because buildevents needs a private secret to pass the honeycomb API Key, so will fail on forks 👍

Reply:
:+1: Removed